src/pages/Search/SearchFiltersCategoryPage.tsx

```diff
@@ -0,0 +1,96 @@
+import React, {useMemo} from 'react';
+import {View} from 'react-native';
+import {useOnyx} from 'react-native-onyx';
+import FullPageNotFoundView from '@components/BlockingViews/FullPageNotFoundView';
+import type {FormOnyxValues} from '@components/Form/types';
+import HeaderWithBackButton from '@components/HeaderWithBackButton';
+import ScreenWrapper from '@components/ScreenWrapper';
+import SelectionList from '@components/SelectionList';
+import RadioListItem from '@components/SelectionList/RadioListItem';
+import useLocalize from '@hooks/useLocalize';
+import useThemeStyles from '@hooks/useThemeStyles';
+import localeCompare from '@libs/LocaleCompare';
+import * as SearchActions from '@userActions/Search';
+import ONYXKEYS from '@src/ONYXKEYS';
+
+function SearchFiltersStatusPage() {
+    const styles = useThemeStyles();
+    const {translate} = useLocalize();
+
+    const [searchAdvancedFiltersForm] = useOnyx(ONYXKEYS.FORMS.SEARCH_ADVANCED_FILTERS_FORM);
+    const activeItems = searchAdvancedFiltersForm?.category;
+    const policyId = searchAdvancedFiltersForm?.policyId ?? '-1';
+
+    const [callCategories] = useOnyx(ONYXKEYS.COLLECTION.POLICY_CATEGORIES);
+    const [singlePolicyCategories] = useOnyx(`${ONYXKEYS.COLLECTION.POLICY_CATEGORIES}${policyId}`);
+
+    let categories: string[] = [];
+    if (!singlePolicyCategories) {
+        categories = Object.values(callCategories ?? {})
+            .map((value) => Object.values(value ?? {}))
+            .map((value2) => value2.map((value3) => value3?.name))
```

Comment:
you need to use readable and reasonable names for specific values when you are working on them.
naming like `value`, `value2`, `value3` tells me nothing and I'm kinda not sure what we need 2 calls to map here.

Please rename so that these value-names mean something, then I can recheck if this logic cannot be written in a simpler way